### ISSUE TITLE
Opinions on default 403 server page accessibility

### ISSUE THREAD

**Issue by jamesjacobs** [w3c/wcag#2164]
Hi all, looking for guidance on the accessibility of default server generated pages, such as the 403 forbidden page. 

Obviously it's commonplace to build and style 404 not found pages to be accessible and fit in with the style of the rest of the site but do people generally go as far to do this to more obscure pages such as 403?  Do people audit these pages? Are these pages also covered by WCAG - I'm assuming they are as they are, at the end of the day, html web pages served to a browser and consumed by a user. I'm interested specifically in the 403 page as a user should never come across this page in their normal usage of a site.

Appreciate it's a bit of an obscure question but it's come up recently and has been flagged for one of my sites after being crawled by a third party automated checker, which was lead to the 403 page after some automated IP blacklisting.

**Comment by patrickhlauke** [comment]
Well yes, anything sent to the user/that the user may interface with should be accessible. If you do know that there's a chance these default error pages will be generated and presented, then it would be a good idea to include those in any accessibility testing for a site.

**Comment by jamesjacobs** [comment]
Thanks Patrick - I thought as much but sometimes it's just useful to sanity check with others when you're a team of 1.

**Comment by mraccess77** [comment]
Yes, but I'd probably consider that they were not part of a set of pages when looking for consistent navigation, identification, multiple ways etc. as this may not be achievable.
